src/Application/Actions/Donations/Create.php

```diff
@@ -91,6 +91,13 @@ protected function action(): Response
                     // See https://stripe.com/docs/api/payment_intents/object
                     'amount' => (100 * $donation->getAmount()),
                     'currency' => 'gbp',
+                    'metadata' => [
+                        'campaignId' => $donation->getCampaign()->getSalesforceId(),
```

Comment:
@tbg-jasonfung I think campaign name as mentioned on the description would also be handy.

Thinking further it's probably good to include all 4 (charity + campaign) (names + IDs) and comms flags so we can see these all at a glance.

Reply:
@NoelLH I don't think we're sending campaign name through the API, should we start doing this?

Reply:
@tbg-jasonfung we are storing the name, have a look at the model. You may need to add a getter if this is the first use. IDEs can generally generate this for you.